Makefile

```diff
@@ -185,7 +185,7 @@ $(foreach DB,$(DATABASES),$(eval $(call database,$(DB))))
 
 ARTIX_PARTS=artix7_50t artix7_200t
 ZYNQ_PARTS=zynq7010
-KINTEX_PARTS=
+KINTEX_PARTS=kintex7_325t
```

Comment:
I've been lurking here and am interested in helping. I wonder if this line is breaking the run for xc7k70tfbg676-2 parts? Admittedly though, it's not obvious to me how runs for Kintex parts ever happen with KINTEX_PARTS= in that file on master.